src/utilities/generateMeta.ts

```diff
@@ -19,16 +19,16 @@ const getImageURL = (image?: Media | Config['db']['defaultIDType'] | null) => {
   return url
 }
 
-export const generateMeta = async (args: {
-  doc: Partial<Page> | Partial<Post>
+export const generateMetaPage = async (args: {
```

Comment:
super-nit: `generateMetaForPage`? I started to wonder what a meta-page was :D

Reply:
it's like soup or salad being confused for super salad. I'll update it